splunk_add_on_ucc_framework/templates/oauth.template

```diff
@@ -22,6 +22,7 @@ logger = log.Logs().get_logger('{{app_name|lower}}_rh_oauth2_token')
 # Map for available proxy type
 _PROXY_TYPE_MAP = {
     'http': socks.PROXY_TYPE_HTTP,
+    # comment the below line if your add-on is not compatible with 'http_no_tunnel' protocol
```

Comment:
Shouldn't we have a config parameter for that?

Reply:
Ideally, requests and httplib2 create a conflict for `import socks` statement, as the one coming from requests doesn't have any `HTTP_NO_TUNNEL` constant. Hence, I've just added it as a code comment in init, otherwise developers usually clean this code up.
Moreover, I think we could clean up this and use requests only as an add-on may use other library and httplib2 just for OAuth setup.